pynest/nest/tests/test_sp/test_all.py

```diff
@@ -54,3 +63,10 @@ def suite():
     nest.set_verbosity('M_WARNING')
     runner = unittest.TextTestRunner(verbosity=2)
     runner.run(suite())
+    # MPI tests
+    if mpi_test:
+        try:
+            call(["mpiexec", "-n", "2", "python", "test_sp/test_issue_578_sp.py"])
+        except:
+            print sys.exc_info()[0]
+            print "Test call with MPI ended in error"
```

Comment:
`print` requires `()` to be Python3 compatible.

And shouldn't this exception propagate so that the test suite knows that the test failed? Currently, failing MPI will be registered as a working test.